app/views/admin/settings/_update_setting_button.html.erb

```diff
@@ -1,3 +1,3 @@
 <% if current_user.has_role?(:super_admin) %>
-  <%= f.submit "Update Settings", class: "crayons-btn mt-4", data: { disable_with: false } %>
+  <%= f.submit "Update Settings", class: "crayons-btn mt-4", aria: { label: defined?(aria_label) ? aria_label : nil }, data: { disable_with: false } %>
```

Comment:
Rails has `local_assigns` for this purpose ([docs](https://api.rubyonrails.org/classes/ActionView/Base.html)). It returns a hash so you get the same behavior in a more succinct way:

```suggestion
  <%= f.submit "Update Settings", class: "crayons-btn mt-4", aria: { label: local_assigns[:aria_label] }, data: { disable_with: false } %>
```

Reply:
Thanks for mentioning this @citizen428. TIL